app/bundles/CoreBundle/Translations/en_US/messages.ini

```diff
@@ -110,6 +110,8 @@ mautic.core.config.form.transifex.password="Transifex password"
 mautic.core.config.form.transifex.password.tooltip="Set the password for your Transifex account"
 mautic.core.config.form.transifex.username="Transifex username"
 mautic.core.config.form.transifex.username.tooltip="Set the username for your Transifex account"
+mautic.core.config.form.dev.hosts="Development hosts"
+mautic.core.config.form.dev.hosts.tooltip="Explicitely whitelist the hosts allowed to request the dev environment using index_dev.php"
```

Comment:
The key should be `mautic.core.config.form.dev_hosts` and `mautic.core.config.form.dev_hosts.tooltip`